service.yml

```diff
@@ -7,10 +7,18 @@ semaphore:
   enable: true
   execution_time_limit: {"hours": 1}
   pipeline_type: cp-dockerfile
-  docker_repos: ['confluentinc/cp-base-new','confluentinc/cp-base-lite','confluentinc/cp-jmxterm']
+  docker_repos: ['confluentinc/cp-base-new', 'confluentinc/cp-base-lite', 'confluentinc/cp-jmxterm']
   build_arm: true
   os_types: ["ubi8"]
   nano_version: true
   maven_skip_deploy: false
   pip_install_package: 'tox==3.28.0'
-
+code_artifact:
+  enable: true
+  package_paths:
+    - maven-snapshots/maven/io.confluent/cp-jmxterm
```

Comment:
is this path same for nightlies as well as RC builds? Can you please check with owner of `cp-dockerfile` pipeline_type for this?

Reply:
> is this path same for nightlies as well as RC builds?
These paths are at the repo level, as long as the builds are in this repo, they will be allowed to write to these paths.